added product group and other contacts (#444)
BCDevOps/devops-requests: .github/ISSUE_TEMPLATE/openshift_project_request.md

* Product Short Name: 
* Product Full Name: 
* Product Description: 

> * Product Group Name (if exists, used to group together OpenShift Project Sets that serve as one product in general): 

Reviewer: ```suggestion
* Product Group Name (if exists, used to group together OpenShift Project Sets that are related to each other): 
```

  - Organization Code:
  - Organization Unit Code: 
  - (refer to https://dir.gov.bc.ca/)
* Other Critical Contacts

>   - (this could be a MISO, Architecture team member, Privacy Specialist, etc)

Reviewer: ```suggestion
  - (this could be a member of IMB, Architecture, Privacy, etc groups)
```
